packages/core/src/delegation/DelegationNode.spec.ts

```diff
@@ -26,9 +26,9 @@ let nodes: Record<string, DelegationNode> = {}
 
 jest.mock('./DelegationNode.chain', () => ({
   getChildren: jest.fn(async (node: DelegationNode) =>
-    node.childrenIds.map((id) => nodes[id] || null)
+    node.childrenIds.map((id) => (id in nodes ? nodes[id] : null))
```

Comment:
Here `nodes` is a Record of string to object, so TS does not expect `nodes[id]` to be undefined. Modifying the type of `nodes` to be nullable results in a ton of unchecked access errors in this test, so I opted for the smaller set of fixes.